klangbecken.py

```diff
@@ -167,8 +167,15 @@ def mutagen_tag_analyzer(playlist, fileId, ext, file_):
 
 
 def ffmpeg_audio_analyzer(playlist, fileId, ext, file_):
+    # To make sure that we find the correct cue_out point, we append ~0.2 seconds
+    # of silence to the end of the track. This guarantees that we always find at
+    # least one silence period.
```

Comment:
I assume you're referring to `apad` here. Could you clarify by mentioning it? :-)